packages/@internationalized/number/src/NumberParser.ts

```diff
@@ -255,7 +255,14 @@ const pluralNumbers = [
 
 function getSymbols(locale: string, formatter: Intl.NumberFormat, intlOptions: Intl.ResolvedNumberFormatOptions, originalOptions: Intl.NumberFormatOptions): Symbols {
   // formatter needs access to all decimal places in order to generate the correct literal strings for the plural set
-  let symbolFormatter = new Intl.NumberFormat(locale, {...intlOptions, minimumSignificantDigits: 1, maximumSignificantDigits: 21});
+  let symbolFormatter = new Intl.NumberFormat(locale, {...intlOptions,
+    // Resets so we get the full range of symbols
+    minimumSignificantDigits: 1,
+    maximumSignificantDigits: 21,
+    roundingIncrement: 1,
+    roundingPriority: 'auto',
+    roundingMode: 'halfExpand'
```

Comment:
Assuming that none of these options affect the result? e.g. if the user passed in different values in `intlOptions` this would override.

Reply:
Right, this is only to get possible symbols. By overriding these with the defaults we get possibly a few extra symbols that aren't possible with the users options, but that won't hurt anything.

I do not believe that any combination of these three options will add additional symbols. Though it is an assumption that I've made based on:  it has worked like this for 4 years before Chrome added these values into the resolvedOptions.